scout/image-details-view.md

```diff
@@ -17,6 +17,8 @@ perspectives. The view displays vulnerabilities and packages that an image
 contains. You can choose whether you want to see data for the image as a whole,
 or for a specific base image or layer.
 
+<!-- TODO: Update image with site link -->
```

Comment:
👀

Reply:
@aevesdocker Yeah, I left them on purpose, but I think I can now update them

Reply:
Actually this is one I can't update yet until DD updates…